package.json

```diff
@@ -8,14 +8,19 @@
   },
   "version": "0.5.0",
   "main": "./lib/server.js",
-  "browser": {
-    "main": "./lib/browser.js"
-  },
+  "browser": "./lib/browser.js",
   "dependencies": {
-    "browserchannel": "^2.0",
-    "through": "^2.3.4"
+    "browserchannel": "^2.0"
+  },
+  "devDependencies": {
+    "np": "^7.6.3"
```

Comment:
This isn't directly used by this package at the moment, so no need to directly include it